### PR TITLE
New version: Rem0o.FanControl version 185

### PR DESCRIPTION
**Pull request by SpecterShell**
Checklist for Pull Requests
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Is there a linked Issue?

Manifests
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/146126)

### CODE REVIEW

File: manifests/r/Rem0o/FanControl/185/Rem0o.FanControl.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+PublisherUrl: https://github.com/Rem0o
+PublisherSupportUrl: https://github.com/Rem0o/FanControl.Releases/issues
+# PrivacyUrl:
+Author: Rem0o
+PackageName: FanControl
+PackageUrl: https://getfancontrol.com/
+License: Freeware
+LicenseUrl: https://github.com/Rem0o/FanControl.Releases/blob/master/LICENSE
+Copyright: © 2024 Rémi Mercier. All rights reserved.
+# CopyrightUrl:
```
Comment (lm-gh) [inline manifests/r/Rem0o/FanControl/185/Rem0o.FanControl.locale.en-US.yaml:17]: `CopyrightUrl: https://github.com/Rem0o/FanControl.Releases/blob/V185/LICENSE`

File: manifests/r/Rem0o/FanControl/185/Rem0o.FanControl.locale.zh-CN.yaml [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+PublisherUrl: https://github.com/Rem0o
+PublisherSupportUrl: https://github.com/Rem0o/FanControl.Releases/issues
+# PrivacyUrl:
+Author: Rem0o
+PackageName: FanControl
+PackageUrl: https://getfancontrol.com/
+License: 免费软件
+LicenseUrl: https://github.com/Rem0o/FanControl.Releases/blob/master/LICENSE
+Copyright: © 2024 Rémi Mercier. All rights reserved.
+# CopyrightUrl:
```
Comment (lm-gh) [inline manifests/r/Rem0o/FanControl/185/Rem0o.FanControl.locale.zh-CN.yaml:17]: `CopyrightUrl: https://github.com/Rem0o/FanControl.Releases/blob/V185/LICENSE`

**Review by OfficialEsco — APPROVED** [review]
@lm-gh just submit a PR yourself, quicker that way =)

**Review by lm-gh — APPROVED** [review]

File: manifests/r/Rem0o/FanControl/185/Rem0o.FanControl.locale.zh-CN.yaml (hunk 2 above)
Comment (lm-gh) [reply manifests/r/Rem0o/FanControl/185/Rem0o.FanControl.locale.zh-CN.yaml:17]: will do in a separate PR

File: manifests/r/Rem0o/FanControl/185/Rem0o.FanControl.locale.en-US.yaml (hunk 1 above)
Comment (lm-gh) [reply manifests/r/Rem0o/FanControl/185/Rem0o.FanControl.locale.en-US.yaml:17]: will do in a separate PR

**Review by lm-gh — APPROVED** [review]

**Comment by lm-gh** [comment]
I will add
`CopyrightUrl: https://github.com/Rem0o/FanControl.Releases/blob/V185/LICENSE`
in a separate PR.
fyi: @OfficialEsco

**Comment by OfficialEsco** [comment]
If only this PR would merge it WOULD be faster.. :D

**Comment by SpecterShell** [comment]
> I will add `CopyrightUrl: https://github.com/Rem0o/FanControl.Releases/blob/V185/LICENSE` in a separate PR. fyi: @OfficialEsco

Added

**Review by OfficialEsco — APPROVED** [review]

**Pull request merged**